frontend/public/components/checkbox.tsx

```diff
@@ -1,21 +1,18 @@
 import * as React from 'react';
+import { Checkbox as PFCheckbox } from '@patternfly/react-core';
 
-export const Checkbox: React.SFC<CheckboxProps> = ({ name, label, checked, onChange }) => (
+export const Checkbox: React.FC<CheckboxProps> = ({ name, label, checked, onChange }) => (
```

Comment:
we should avoid using React.FC for new code per [CONSOLE-4630](https://issues.redhat.com/browse/CONSOLE-4630)

```suggestion
export const Checkbox = ({ name, label, checked, onChange }: CheckboxProps) => (
```

Reply:
Thanks, @logonoff.  I missed that.  We should mention that to everyone at scrum again.

Reply:
Fixed.